src/cgame/cg_main.cpp

```diff
@@ -886,6 +893,12 @@ cvarTable_t cvarTable[] = {
     {&etj_speedColorUsesAccel, "etj_speedColorUsesAccel", "0", CVAR_ARCHIVE},
     {&etj_speedAlign, "etj_speedAlign", "0", CVAR_ARCHIVE},
 
+    {&etj_drawAccel, "etj_drawAccel", "1", CVAR_ARCHIVE},
+    {&etj_accelX, "etj_accelX", "320", CVAR_ARCHIVE},
+    {&etj_accelY, "etj_accelY", "350", CVAR_ARCHIVE},
+    {&etj_accelSize, "etj_accelSize", "3", CVAR_ARCHIVE},
+    {&etj_accelAlign, "etj_accelAlign", "0", CVAR_ARCHIVE},
```

Comment:
This isn't implemented at all?

Reply:
whoopsie, copy-pasted and forgot to replace